apps/api/src/server/app/inspector/appeal-formatter.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash';
```

Comment:
I think should be lodash_es. And we should refactor all of them that are still using lodash.

Reply:
how come? i saw Ben using lodash_es as well

Reply:
`lodash-es` is the esm version and even if the cjs works it is better to use the native version. Also the FE might use it in the future and it will help with bundle size.